### DISCUSSION TITLE
Tailwind CSS with vite

### DISCUSSION THREAD

**Discussion by itspriyambhattacharya** [tailwindlabs/tailwindcss#7270] [Help]
I am making my multi-page portfolio with tailwind css and vite. Everything is fine in index.html (home page). But in my about page, I can only use the classes which I have already used in index.html.

Eg. If I have used w-12 in home page, I can only use w-12 in my about page. If I am using any other value say, w-14, it is not accepting. I cannot even use w-[123px].
Please Suggest me what to do.

Folder Structure Image Below
![file](https://user-images.githubusercontent.com/84953701/151751459-f2c152e1-d29c-4485-a2a0-7ab9781fea69.png)

**Comment by ExEr7um** [comment]
Can you show your `tailwind.config.js`?

**Reply by itspriyambhattacharya** [reply]
module.exports = {
  content: ["*"],
  theme: {
    extend: {
      colors: {
        'primary': '#AC00FD',
        'bg': '#F7F7F7'
      },
      fontFamily: {
        'roboto': ['Roboto', 'sans-serif']
      }
    },
  },
  plugins: [],
}
![tailwind](https://user-images.githubusercontent.com/84953701/151919012-c370a873-8fdf-4598-a1f6-a9c0d8796a3b.png)

**Reply by ExEr7um** [reply]
Change your `content` to
```javascript
content: ['./**/*.{html}']
```

**Reply by itspriyambhattacharya** [reply]
Thank You Sir, It Worked. 
Can you please explain what was my mistake and what does ./**/*.html mean?
It would be really helpful for me.

**Reply by ExEr7um** [reply]
To generate your CSS Tailwind first need to know what classes are you using. In `content` you specify in which files it should look for them.
In your solution `[*]` it was looking for every single file inside your root directory.
In my solution it also looks inside every nested folder. That's what `/**/*` means. And `.html` means that it should only look inside HTML files.

**Reply by itspriyambhattacharya** [reply]
Thank You Sir for your valuable time.
